styles.css

```diff
@@ -212,3 +202,287 @@
 .media-link:hover {
   color: aqua;
 }
+
+.Introduction {
+  background-color: #fff;
+  position: absolute;
+  top: 65px;
+  margin: 0;
+  border: 0;
+  display: block;
+  width: 100%;
+  height: 800px;
+  border-radius: 0 0 0 75px;
+}
+
+.sample-project {
+  position: absolute;
+  top: 950px;
+  background-color: #fff;
+  height: 680px;
+  margin: 100px 24px;
+  border-radius: 16px;
+  display: grid;
+  gap: 1px;
+  grid-template-areas: 'header header header' 'description description description' 'footer footer footer';
+}
```

Comment:
- [ ] Kindly make sure to use the grid for a container of the project and it will be better if you don't use absolute values which you get from Figma rather than use grid for the container and flexbox for content for this project

Reply:
@omarsalem7 
Hi, Thanks for the review. I did apply a grid for the project container and a flexbox for the contents inside. I didn't get the suggestion. Can you elaborate?